docs/CLI/authentication.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Authentication
-sidebar_position: 1
+sidebar_position: 10
```

Comment:
Hello @TerryHowe,

Could you please explain the reasons for changing these positions to tens of their original number?

Reply:
Normally when I've used these position type constructs, I've left room in the middle to insert new things or change the order without changing everything. I assume this works the same with docusaurus

Reply:
> Normally when I've used these position type constructs, I've left room in the middle to insert new things or change the order without changing everything. I assume this works the same with docusaurus

Makes sense, and yes it works the same in Docusaurus as well.